casr/src/triage.rs

```diff
@@ -77,7 +70,26 @@ impl<'a> CrashInfo {
 
         let mut casr_cmd = Command::new(tool);
         casr_cmd.args(&args);
-        casr_cmd.envs(envs);
+
+        // Add envs
+        if self
+            .target_args
+            .iter()
+            .any(|x| x.contains("-detect_leaks=0"))
+        {
+            let asan_options = std::env::var("ASAN_OPTIONS").unwrap_or(String::new());
+            casr_cmd.env(
+                "ASAN_OPTIONS",
+                if asan_options.is_empty() {
+                    "detect_leaks=0".to_string()
+                } else {
+                    format!("{asan_options},detect_leaks=0",)
+                },
+            );
+        }
+        if self.casr_tool.ends_with("casr-python") {
+            casr_cmd.env("LD_PRELOAD", get_atheris_lib()?);
```

Comment:
This requires invoking python and can be slow. Can we save atheris lib path once somewhere?

Reply:
I added field `envs` to `CrashInfo` struct